examples/customs items/create_customs_items.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+require 'easypost'
+
+EasyPost.api_key = ENV['EASYPOST_TEST_API_KEY']
+
+customs_item = EasyPost::CustomsItem.create(
```

Comment:
I believe these parameters need to be wrapped in a dictionary. This function takes a dictionary, not specific parameters.